pygmt/x2sys.py

```diff
@@ -269,8 +269,8 @@ def x2sys_cross(tracks=None, outfile=None, **kwargs):
         - **l** sets lower speed [Default is 0].
         - **u** sets upper speed [Default is Infinity].
         - **h** does not limit the speed but sets a lower speed below which \
-        headings will not be computed (i.e., set to NaN) [Default calculates \
-        headings regardless of speed].
+          headings will not be computed (i.e., set to NaN) [Default \
+          calculates headings regardless of speed].
```

Comment:
I think the newline is unnecessary.
```suggestion
          headings will not be computed (i.e., set to NaN) [Default
          calculates headings regardless of speed].
```